### ISSUE TITLE
Date format changed in csse_covid_19_daily_reports/04-06-2020.csv file in the last update

### ISSUE THREAD

**Issue by arjunachala** [CSSEGISandData/COVID-19#2070]
COVID-19/csse_covid_19_data/csse_covid_19_daily_reports/04-06-2020.csv date format changed to mm/dd/yy is yyyy-mm-dd in all other files

**Comment by jamescowens** [comment]
Yep. I had to change my custom C++ filter to deal with that stuff. They seem to be flipping back and forth between date formats.

**Comment by Lucas-Czarnecki** [comment]
Date/time format is an ongoing issue. I maintain a tidy repo [HERE](https://github.com/Lucas-Czarnecki/COVID-19-CLEANED-JHUCSSE) that addresses these kinds of inconsistencies.

**Comment by vince-angelo** [comment]
The FIPS codes for counties with leading zeros are also truncated to four places (e.g., "8001" instead of "08001" for Adams, Colorado on line 10) in that same adjusted file.

**Comment by pavelfomin** [comment]
Several older files have been updated with a new date format also (i.e. `3/29/20 23:08`). However, the new files 04-07-2020.csv and 04-08-2020.csv have the original date format.

**Comment by cipriancraciun** [comment]
This is not the first time dates or location names are inconsistent.  (And perhaps it will continue to be so.)  The only sane approach is to either used a derived dataset, or implement your loading code to be more robust and patch over these issues.

For those interested I've published a derived dataset that I've described in #1281 and available at: https://github.com/cipriancraciun/covid19-datasets

**Comment by jamescowens** [comment]
My C++ filter deals with that...
